receiver/otlpreceiver/otlp_test.go

```diff
@@ -941,10 +961,13 @@ func newGRPCReceiver(t *testing.T, endpoint string, tc consumer.Traces, mc consu
 	return newReceiver(t, factory, cfg, otlpReceiverID, tc, mc)
 }
 
-func newHTTPReceiver(t *testing.T, endpoint string, tc consumer.Traces, mc consumer.Metrics) component.Component {
+func newHTTPReceiver(t *testing.T, endpoint string, tracesURLPath string, metricsURLPath string, tc consumer.Traces, mc consumer.Metrics) component.Component {
 	factory := NewFactory()
 	cfg := factory.CreateDefaultConfig().(*Config)
 	cfg.HTTP.Endpoint = endpoint
+	cfg.HTTP.TracesURLPath = tracesURLPath
+	cfg.HTTP.MetricsURLPath = metricsURLPath
+	cfg.HTTP.LogsURLPath = defaultLogsURLPath
```

Comment:
Why is logs different here?

Reply:
I've changed the log path difference so it doesn't cause any confusion.